project/plugins.sbt

```diff
@@ -8,8 +8,8 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-less" % "1.1.0")
 addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.4")
 
 addSbtPlugin("com.gu" % "sbt-teamcity-test-reporting-plugin" % "1.5")
-addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "0.9.4")
+addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.12")
 
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.6.1")
 
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
\ No newline at end of file
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
```

Comment:
fyi if you update to SBT 1.4 or later (or it might be 1.5) then you can remove this unmaintained plugin, as the things needed for snyk are built in.